OpenSim/Simulation/Model/Station.h

```diff
@@ -37,38 +36,56 @@ class Body;
 //=============================================================================
 //=============================================================================
 /**
- * A class implementing a Station. A Station is a point fixed to and defined with 
- * respect to a reference frame. The reference frame can be a Body (fixed 
- * to origin of a Body), a FixedFrame (affixed to another PhysicalFrame) or any 
- * other PhysicalFrame. The main functionality provided by Station is to find its 
- * location in any Frame.
+ * A Station is a Point fixed to and located on a PhysicalFrame, which can be a 
+ * Body (w.r.t. the Body origin), Ground or any PhysicalOffsetFrame.
+ * A Station is an analogous physical Point to a PhyscialFrame where joints,
+ * constraints and forces can be attached and/or applied.
```

Comment:
This sentence is confusing to me. Consider: "Stations are similar to PhysicalFrames in that constraints and forces can be specified between stations."